src/VisualStudio/Core/SolutionExplorerShim/AnalyzersCommandHandler.cs

```diff
@@ -303,6 +311,23 @@ private void SetSeverityHandler(object sender, EventArgs args)
             }
         }
 
+        private void OpenDiagnosticHelpLinkHandler(object sender, EventArgs e)
+        {
+            if (_tracker.SelectedDiagnosticItems.Length != 1 ||
+                string.IsNullOrWhiteSpace(_tracker.SelectedDiagnosticItems[0].Descriptor.HelpLinkUri))
+            {
+                return;
+            }
+
+            string link = _tracker.SelectedDiagnosticItems[0].Descriptor.HelpLinkUri;
```

Comment:
No, it does not. We don't even show the menu item unless the diagnostic specifies a help link.

I could do this, however. I would rename the menu item to "View/Find Help..." and show it all the time; if no help link is specified we would fall back to a Bing search.